partner_industry_secondary/migrations/11.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_column_renames = {
+    'res_partner': [('sector_id', None)],
+}
+
+_field_adds = [
+    ('parent_id', 'res.partner.industry', 'res_partner_industry', 'many2one',
+     'int4', 'partner_industry_secondary'),
+    ('secondary_industry_ids', 'res.partner.industry', 'res_partner_industry',
+     'many2many', False, 'partner_industry_secondary'),
+]
+
+_xmlid_renames = [
+    ('partner_sector.group_use_partner_sector_for_person',
+     'partner_industry_secondary.group_use_partner_industry_for_person'),
+]
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    cr = env.cr
+    if openupgrade.table_exists(cr, 'res_partner_sector'):
+        openupgrade.add_fields(env, _field_adds)
+        openupgrade.rename_columns(cr, _column_renames)
+        openupgrade.rename_xmlids(cr, _xmlid_renames)
+        cr.execute(
+            """
+            ALTER TABLE res_partner_industry
+            ADD COLUMN %s integer;
+            """ % openupgrade.get_legacy_name('old_id')
```

Comment:
A better name can be `old_sector_id` (combined with the legacy name)

Reply:
Aesthetics  😏 But I like your suggestion.